src/content/1.7/modules/concepts/pdf.md

```diff
@@ -0,0 +1,211 @@
+
+You can alter the Core files OR override them by copying them on `themes/<your-themes>/pdf` folder.
+
+{{% notice warning %}}
+It's not possible to override a PDF template in a module.
+{{% /notice %}}
+
+#### Delivery Slip
+
+Delivery Slips are available in `Orders > Delivery Slips` section of the Back Office.
+
+The following files are used to generate the PDF file:
+
+```
+pdf
+├── delivery-slip.addresses-tab.tpl
+├── delivery-slip.payment-tab.tpl
+├── delivery-slip.product-tab.tpl
+├── delivery-slip.style-tab.tpl
+├── delivery-slip.summary-tab.tpl
+└── delivery-slip.tpl
+```
+
+#### Invoices
+
+Invoices are available in `Orders > Invoices` section of the Back Office and in `Orders List` section of the Customer Back Office.
+
+The following files are used to generate the PDF file:
+
+```
+pdf
+├── invoice.addresses-tab.tpl
+├── invoice-b2b.tpl
+├── invoice.note-tab.tpl
+├── invoice.payment-tab.tpl
+├── invoice.product-tab.tpl
+├── invoice.shipping-tab.tpl
+├── invoice.style-tab.tpl
+├── invoice.summary-tab.tpl
+├── invoice.tax-tab.tpl
+├── invoice.total-tab.tpl
+└── invoice.tpl
+```
+
+#### Merchandise Returns
+
+Order (or Merchandise) Returns are available in `Orders > Merchandise Returns` section of the Back Office.
+
+The following files are used to generate the PDF file:
+
+```
+pdf
+├── order-return.addresses-tab.tpl
+├── order-return.conditions-tab.tpl
+├── order-return.product-tab.tpl
+├── order-return.summary-tab.tpl
+└── order-return.tpl
+```
+
+#### Order Slips
+
+Order Slips are available in `Orders > Order Slips` section of the Back Office.
+
+```
+pdf
+├── order-slip.payment-tab.tpl
+├── order-slip.product-tab.tpl
+├── order-slip.summary-tab.tpl
+├── order-slip.total-tab.tpl
+└── order-slip.tpl
+```
+
+#### Supply Orders
+
+Since 1.7, Supply Orders are not available anymore in the Back Office but you can still "trigger" the generation of the PDF files.
+
+```
+pdf
+├── supply-order.addresses-tab.tpl
+├── supply-order-footer.tpl
+├── supply-order-header.tpl
+├── supply-order.product-tab.tpl
+├── supply-order.tax-tab.tpl
+├── supply-order.total-tab.tpl
+└── supply-order.tpl
+```
+
+#### Common Templates
+
+These files are used by most of the previous files:
+
+```
+pdf
+├── footer.tpl
+├── header.tpl
+└── pagination.tpl
+```
+
+#### Minimal Exemple: customize design of the Invoices
+
+```smarty
+{* /themes/your-theme/pdf/invoice.style-tab.tpl *}
+{assign var=color_header value="#25B9D7"}
+{assign var=color_border value="#3ED2F0"}
+{assign var=color_border_lighter value="#DFF5F9"}
+{assign var=color_line_even value="#FAFBFC"}
+{assign var=color_line_odd value="#6C868E"}
+{assign var=font_size_text value="12pt"}
+{assign var=font_size_header value="12pt"}
+{assign var=font_size_product value="12pt"}
+{assign var=height_header value="25px"}
+{assign var=table_padding value="7px"}
+```
+
+You will get an Invoice would look like this:
+
+{{< figure src="../img/invoice_style.png" title="Stylized Invoice exemple" >}}
```

Comment:
```suggestion
{{< figure src="../img/invoice-style.png" title="Stylized Invoice exemple" >}}
```
Avoid underscore for SEO purpose

Reply:
Nani ? :neutral_face:

Reply:
Yes. Apparently (I did not know this morning) SEO is better for dashes than for underscores. And we want the devdocs, images included, to be SEO boosted in order to make it easier for developers to find their answers when googling an issue.

You can check it 😉

Reply:
I don't disagree on this, but it's not a schema it's a screenshot.

Anyway, I'll update.

Can I have a review on the content now, please ? 🙏